README.md

```diff
@@ -6,9 +6,13 @@
 </a>
 
 # cosima-recipes
-Example recipes using the [`cosima-cookbook`](https://github.com/COSIMA/cosima-cookbook) infrastructure.
+This is a collection of example recipes and tutorials on analysing Ocean and SeaIce model output produced by [COSIMA: Consortium for Ocean-Sea Ice Modelling in Australia](http://cosima.org.au/) and its members. Most examples use output from the [ACCESS-OM2 model](https://doi.org/10.5194/gmd-13-401-2020), while some also use results from MOM6 configurations and remote sensing observations. To access the data used in these recipes you need an account and project with the Australian based [National Computational Infrastructure (NCI)](https://nci.org.au/)
 
-To **Get Started** with the `cosima-cookbook`, clone this repository locally, probably best in your local space on one of the NCI HPC machines so you can have access to model output. The repository includes a bunch of examples with which you can begin to construct your own analysis code.
+To **get started** with `cosima-recipes`, clone this repository in your local space on one of the NCI HPC machines so you can have access to model output. You should then be able to run these recipes through an [Australian Research Environment (ARE)](https://are.nci.org.au/) JupyterLab session running python. You need to join projects _hh5, xp65, ik11, cj50_ and _ol01_ to run the recipes and acccess the data analysed.
+
+When starting ARE, include the projects in the _Storage_ line: _gdata/xp65+gdata/ik11+gdata/cj50+gdata/hh5+gdata/oi10_ as well as your own project. In _Module directories_, set _/g/data/hh5/public/modules_ and in _Modules_ set _conda/analysis3_.
```

Comment:
Add note about instance size

Reply:
Shouldn't we use `this format` for input the users should put enter?

Reply:
Good point - yes :)